src/System/Random/Internal.hs

```diff
@@ -241,6 +245,25 @@ class Monad m => StatefulGen g m where
 
 
 
+-- | This class is designed for stateful RNGs that can be saved as immutable data type.
```

Comment:
```suggestion
-- | This class is designed for stateful pseudo-random number generators that
-- can be saved as and restored from an immutable data type.
```

I don't want there to be confusion between "RNG" and "PRNG"; the non-abbreviated version "pseudo-random number generator" is used throughout.